src/jit/lsra.cpp

```diff
@@ -5913,9 +5913,25 @@ bool LinearScan::checkActiveInterval(Interval* interval, LsraLocation refLocatio
             else
             {
                 RefPosition* nextAssignedRef = recentAssignedRef->nextRefPosition;
+#ifdef _TARGET_ARM_
+                // This function is invoked only from allocateBusyReg() through checkActiveIntervals().
+                //
+                // For ARM32, when we try to allocate a double register which consists of two float registers,
+                // tryAllocateFreeReg() may fail to allocate a double register even if one of two float
+                // reigsters is assigned to a inactive constant interval.
+                // https://github.com/dotnet/coreclr/pull/14080
+                //
+                // Therefore an inactive constant interval can be tested here.
```

Comment:
This should probably be worded: "Therefore an inactive constant interval may be encountered here."
Otherwise looks good.

Reply:
Thanks !